game_eggs/steamcmd_servers/ark_survival_ascended/egg-ark--survival-ascended.json

```diff
@@ -0,0 +1,132 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2023-10-28T14:43:36-04:00",
+    "name": "Ark: Survival Ascended",
+    "author": "blood@darkartsgaming.com",
+    "description": "ARK is reimagined from the ground-up into the next-generation of video game technology with Unreal Engine 5! Form a tribe, tame & breed hundreds of unique dinosaurs and primeval creatures, explore, craft, build, and fight your way to the top of the food-chain. Your new world awaits!",
+    "features": null,
```

Comment:
i would add here:

```
"features": [
        "steam_disk_space"
    ],
```

its only possible, when you edit egg oputside ptero

Reply:
done. i was wondering how to do that from inside the panel

Reply:
> done. i was wondering how to do that from inside the panel

not possible yet. But maybe sometimes :D